servicemesh-release-notes/topics/rn-known-issues.adoc

```diff
@@ -28,12 +28,12 @@ https://github.com/istio/old_issues_repo/issues/115
 ////
 * The istio-init container requires a privileged security context or at least to run as root and to have the NET_ADMIN capability.  The istio-init container needs to be privileged because it needs to properly configure the iptables rules in the pod in order to intercept network connections.  The team is currently investigating ideas for ways to reduce the privileges required by Istio.
 
+https://issues.jboss.org/browse/MAISTRA-61[MAISTRA-61] After all applicable resources are deployed to OpenShift, Istio takes several minutes to begin routing to the correct pod.
+
```

Comment:
Could we change this description to be something like the following?

> After all applicable resources are deployed to OpenShift, Istio sidecars may lose information about their routes and can no longer communicate with services until the next update is received

Reply:
@knrc I adopted your wording.